.github/workflows/deploy.yml

```diff
@@ -39,7 +39,7 @@ jobs:
         run: pip install -r requirements.txt
 
       - name: Build site
-        run: mkdocs build
+        run: mkdocs build --strict
```

Comment:
I also added this. Example of why this must be passed for all mkdocs builds: https://cibuildwheel.pypa.io/en/v2.18.0/options/

Reply:
oh wow, that is .... not good